helm/hpcc/templates/_helpers.tpl

```diff
@@ -2086,3 +2086,17 @@ Pass in dict with root
  {{ printf "**** %s: %s ****" (upper $warning.severity) $warning.msg }}
 {{- end -}}
 {{- end -}}
+
+{{- define "hpcc.checkResources" -}}
+{{/*
+Returns true if resources provided, otherwise return false
```

Comment:
This isn't really true.  It returns blank if true, otherwise false.  More normal is to return "true" if true, and blank if false.  (E.g. see "hpcc.hasPlaneForCategory"
Then the result can be tested with

   {{- $hasResources := include "hpcc.checkResources" (dict "resources" $component.resources) -}}
   {{- if not $hasResources -}}